auto-configurations/mcp/spring-ai-autoconfigure-mcp-client-common/src/main/java/org/springframework/ai/mcp/client/common/autoconfigure/properties/McpClientCommonProperties.java

```diff
@@ -103,7 +103,7 @@ public enum ClientType {
 	 * <p>
 	 * This configuration is used to enable or disable tool callbacks in the MCP client.
 	 */
-	private Toolcallback toolcallback = new Toolcallback();
+	private ToolCallback toolCallback = new ToolCallback();
```

Comment:
Spring Boot doesn't recommend to use kebab-case for intermediate configuration parts.

For example: https://github.com/spring-projects/spring-boot/blob/30440c5d937bb19a801eff1cb2974d1074f6f3f6/module/spring-boot-webmvc/src/main/java/org/springframework/boot/webmvc/autoconfigure/WebMvcProperties.java#L99-L103